UDx-container/entrypoint.sh

```diff
@@ -21,10 +21,122 @@
 # command such as bash, make, or g++ to be executed inside this
 # container. 
 
-dir=$1
-cmd=$2
+VSQL=/opt/vertica/bin/vsql
+vsdk_dir=$1
+vsdk_cmd=$2
 
-echo cd \"$dir\"
-echo sh -c \"$cmd\"
-cd "$dir"
-sh -c "$cmd"
+ADMINTOOLS="${VERTICA_OPT_DIR}/bin/admintools"
+
+# Vertica should be shut down properly
+function shut_down() {
+    echo "Shutting Down"
+    vertica_proper_shutdown
+    echo 'Stopping loop'
+    STOP_LOOP="true"
+}
+
+function vertica_proper_shutdown() {
+    db=$(${ADMINTOOLS} -t show_active_db)
+    case "$db"x in
+        x) 
```

Comment:
preferred unix method of checking for an empty string in a case statement:
```
case $db in
    '')
```
But case statements are ugly.  Why not just a `[[ -z $db ]] && return 0`